<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Win</class>
 <widget class="QDialog" name="Win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelIp">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ip</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPort">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIp">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>251</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPort">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2Ok">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>95</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPass">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pass</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPass">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>261</width>
     <height>33</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAuth">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>95</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Auth</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditText">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>541</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>240</y>
     <width>95</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2Text">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>421</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labell">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>                          STATUS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>labelIp</zorder>
  <zorder>labelPort</zorder>
  <zorder>lineEditIp</zorder>
  <zorder>spinBoxPort</zorder>
  <zorder>pushButton_2Ok</zorder>
  <zorder>labelPass</zorder>
  <zorder>lineEditPass</zorder>
  <zorder>pushButtonAuth</zorder>
  <zorder>textEditText</zorder>
  <zorder>pushButtonSend</zorder>
  <zorder>lineEdit_2Text</zorder>
  <zorder>labell</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
